install.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func install() {
+	if os.Getegid() != 0 {
+		fmt.Println("Please run the program with sudo privileges")
+		os.Exit(1)
+	}
+	fmt.Println("Installing ssdeep package...")
+	command := exec.Command("/bin/sh", "-c", "apt-get install ssdeep")
+	if err := command.Run(); err != nil {
```

Comment:
Modify this to install `ssdeep` using the source - this is so it works in all *nix distributions.